mmv1/products/workflows/Workflow.yaml

```diff
@@ -118,6 +118,5 @@ properties:
       Format: projects/{project}/locations/{location}/keyRings/{keyRing}/cryptoKeys/{cryptoKey}
   - !ruby/object:Api::Type::KeyValuePairs
     name: 'userEnvVars'
```

Comment:
We'll need to enable the following tests for ga

For the following you'll need to remove this line for min_beta on the [mmv1 testcase
](https://github.com/ScottSuarez/magic-modules/blob/bb4a38b665058d204355089a478731ad72fb4c31/mmv1/templates/terraform/examples/workflow_beta.tf.erb#L16)

And remove the google-beta references from the test
https://github.com/ScottSuarez/magic-modules/blob/bb4a38b665058d204355089a478731ad72fb4c31/mmv1/templates/terraform/examples/workflow_beta.tf.erb#L8-L9

Same thing for the handwritten test

remove the conditional printing of the full test
https://github.com/ScottSuarez/magic-modules/blob/bb4a38b665058d204355089a478731ad72fb4c31/mmv1/third_party/terraform/services/workflows/resource_workflows_workflow_test.go.erb#L258

and remove the provider=google-beta lines
https://github.com/ScottSuarez/magic-modules/blob/bb4a38b665058d204355089a478731ad72fb4c31/mmv1/third_party/terraform/services/workflows/resource_workflows_workflow_test.go.erb#L216

This is, assuming that they don't have other beta only dependencies.

Reply:
Hi @ScottSuarez, thanks for your review and comments. 
The tested I modified in workflow_basic.tf.erb and resource_workflows_workflow_test.go.erb provides a better test as it uses the declared env vars in the actual workflow. Maybe I should remove the beta test in workflow_beta.tf.erb and resource_workflows_workflow_test.go.erb? 
https://github.com/pikapower/magic-modules/blob/1938e2e78389f754b88eb8d2e3a9db092afe26c9/mmv1/templates/terraform/examples/workflow_basic.tf.erb#L14C1-L16C4
https://github.com/pikapower/magic-modules/blob/1938e2e78389f754b88eb8d2e3a9db092afe26c9/mmv1/templates/terraform/examples/workflow_basic.tf.erb#L32

Reply:
We can remove `workflow_beta` if you like..

We should retrain 
https://github.com/ScottSuarez/magic-modules/blob/bb4a38b665058d204355089a478731ad72fb4c31/mmv1/third_party/terraform/services/workflows/resource_workflows_workflow_test.go.erb#L216

Because it tests update funcitonality.

Reply:
> We should retrain
> https://github.com/ScottSuarez/magic-modules/blob/bb4a38b665058d204355089a478731ad72fb4c31/mmv1/third_party/terraform/services/workflows/resource_workflows_workflow_test.go.erb#L216
> 
> Because it tests update funcitonality.

Please correct me if I'm wrong, doesn't this non-beta update function test everything that the beta version does? I think the beta update function was just a copy of the non-beta version with the added feature of user_env_vars, it was added to the commit just to have it tested for beta. Now that I have the feature included and tested in the non-beta update functions, it should still be tested as well regardless of the ' unless ga' label.
https://github.com/pikapower/magic-modules/blob/bb4a38b665058d204355089a478731ad72fb4c31/mmv1/third_party/terraform/services/workflows/resource_workflows_workflow_test.go.erb#L16

Reply:
You're totally right ! Any oversight on my part. You can remove those two tests if thats your preference :)

Reply:
@ScottSuarez Thanks for the clarification. I just updated the pr to include the change.